src/main/java/racingcar/domain/Position.java

```diff
@@ -1,8 +1,9 @@
 package racingcar.domain;
 
+import java.util.Comparator;
 import java.util.Objects;
 
-public class Position {
+public class Position implements Comparator {
```

Comment:
```suggestion
public class Position implements Comparable<Position> {
```

이렇게 작성하면 별도의 형변환이 필요없습니다. 그리고 자기자신의 정보를 파라미터로 받을 필요도 없겠죠